### ISSUE TITLE
Accessibility feedback for Campaign Landing template, design intent collaboration

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#6973]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## VSA accessibility feedback

1. What are the next steps planned for this work? Possibility of usability testing?
2. **✔️ Consider** for mobile specs.
    a. How the banner graphic will be treated
    b. How the YouTube video and image will be handled (due to potential cell data impact, blind & deafblind user needs and uncertainty of accessibility of video)
    c. Cognitive overload of the maximum page shown
    d. Handling of graphics in the examples under "What you can do"
1. **❗️ Must** — **If** suicide prevention is where the campaign template may first be applied, please have usability testing first. These users may be vulnerable, cognitively. If the content / UI is overwhelming or hard to use, especially on mobile, it may create emotional flooding.
1. **✔️ Consider** requiring authors to provide the link to redirect, as part of the campaign publishing process, along with the end date of the campaign. 
2. **⚠️Should** not use ALL CAPS for the "WHAT YOU CAN DO" slug headings:
    a. Some screen readers read capitalized text letter-by-letter. This occasionally even occurs when the HTML is in sentence-case and the CSS forces the capitalization. VoiceOver is especially persnickety about this.
    b. All capital letters are harder to read (for everyone, but especially people with reading disabilities). The use of All Caps can reduce the readability of your text. Usability expert Jakob Nielsen has found that reading on screen can be around 25% slower than reading from paper, and reading All Caps can be a further 10% slower.
    c. For people with dyslexia, ALL CAPS text may have no difference in shape: all the words are just one big rectangle. Sentence capitals have differentiated shapes based on the letters used. When producing information to be read by others, it is important to remember that up to 10% of your readers may have dyslexia. Dyslexia friendly text will have improved readability and better visual impact for all readers, but especially those with dyslexia.

## VSP accessibility feedback
**❗️ Must** implement for an accessible experience
1. Any PDFs being downloaded must be accessible. This should be planned for sooner than later. Creating PDFs accessibly is more art than science and can be time-consuming.

**⚠️Should** implement for good user experience on mobile and AT
1. Each `panel` should have a good ARIA label to describe it. Screen reader users will appreciate having this extra hook for moving by chunk, not just heading. Using Panel two as an example:
    ```html
    <section aria-label="Mission Act: Why this matters to you">
      ...content
    </section>
    ```
2. Headings should be nested properly. I think some of the panels would benefit from `<h3>` inside your `<h2>` main panel headers.
3. Should have a user experience plan for when campaigns are ended. Will this just be a `301` permanent redirect to a campaign "landing"? Just a redirect to the VA.gov front page? Something else?

**✔️ Consider** for suggestions/enhancements
1. Researching the viability of video transcripts and closed captions. Even if videos go on a service provider, it would be great to have our own custom transcripts and `.VTT` files to upload. AI does a so-so job captioning video, but this is important enough it's worth exploring viability and cost-basis of having our videos professionally transcribed.
2. Have you thought about [lazy loading the images and/or video](https://developers.google.com/web/fundamentals/performance/lazy-loading-guidance/images-and-video) further down the pages? This could be a nice enhancement for mobile users on metered data.
3. I'd like to use the download icon and [guidance around adding file size](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/152#issuecomment-553127340) to the download PDF links. 

## Definition of done
Accessibility specialists (Jennifer or Trevor) will close this ticket when the following items are done:

* Designer(s) acknowledge feedback
* Designer(s) explain in comment on this ticket what decisions were made based on feedback

**Comment by jenniferlee-dsva** [comment]
@jenstrickland Replying to ^:

>❗️ Must implement for an accessible experience
1. Any PDFs being downloaded must be accessible. This should be planned for sooner than later.
Creating PDFs accessibly is more art than science and can be time-consuming.

_[Agree. Note however that PW team will not be responsible for creating PDFs for these pages. Various business stakeolder/authors will be using these, and they are already obligated to adhere to the VA's 508 office guidelines. So for the template product - this is a moot issue, as this template product is not itself creating content or PDFs or videos -- it is simply creating the template AX in Drupal. /Jen]_

>⚠️Should implement for good user experience on mobile and AT
1. Each panel should have a good ARIA label to describe it. Screen reader users will appreciate having this extra hook for moving by chunk, not just heading. Using Panel two as an example:
```html
<section aria-label="Mission Act: Why this matters to you">
  ...content
</section>
```

[Thx for this - it would be great if we can bake this into the CMS template, so that the ARIA label is automatically added as part of content entry for those elements. @johnhashva - this should be included as a technical requirement during Drupal build. (It might be a FE template thing; I'm not sure, so I'd raise it during the Drupal AX build. / Jen]

FYSA ^ @stephorkandatx 
FYSA @rtwell - in case there are design things that need to be addressed. (Like the ALL CAPS supertopic labels? It seems like that could be addressed via alt text or some tag that tells readers to read it as normal words.)

**Comment by johnhashva** [comment]
Re: [this should be included as a technical requirement during Drupal build. (It might be a FE template thing; I'm not sure, so I'd raise it during the Drupal AX build. / Jen]  -- Noted

**Comment by jenstrickland** [comment]
The ALL CAPS labels would be an issue for some screen reader implementations and could provide an aria-label to address, but a more common issue for dyslexic users that labels won't help with. 

Thanks for documenting decisions made, and I look forward to seeing how the work progresses. Let me know if I may be of assistance.

**Comment by jenniferlee-dsva** [comment]
We've discussed the all caps super topic labels before @jenstrickland.Since this is essentially a design element, could the super topic labels for instance be an image with alt text to indicate what the label word is? The section super topic labels are hard set - they do not change.  I leave it to @rtwell, FE, and you to propose a good solution.

**Comment by jenstrickland** [comment]
For dyslexic users the ALL CAPS can be problematic so the guidance is to avoid using them. Adding an aria-label would help screen readers. It's better not to use an image, since the text won't scale for low-vision users.
